tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
 
     def handle_request(self, request, **kwargs):
-
         search_params = {}
-        limit = int(kwargs.get('limit', self.query_limit_default))
-        offset = int(kwargs.get('offset', 0))
+
+        # Capture query bounding parameters (limit/offset does not apply when filtering by object ids)
+        object_ids = kwargs['objectIds'].split(',') if kwargs.get('objectIds') else []
+        limit = 0 if object_ids else int(kwargs.get('limit') or self.query_limit_default)
```

Comment:
It sounds like what you're saying is we never want to limit requests for object ids. That makes sense to me, especially because the content returned will include very low overhead. I'll see what I can do to make that work.